common/views/components/IIIFViewerPrototype/IIIFViewerPrototype.tsx

```diff
@@ -471,7 +471,6 @@ const IIIFViewerPrototype: FunctionComponent<IIIFViewerProps> = ({
           isResizing={isResizing}
           ref={mainAreaRef}
         >
-          {showZoomed && <ZoomedImagePrototype />}
```

Comment:
[The place where we want this is duplicated below](https://github.com/wellcomecollection/wellcomecollection.org/pull/6352/files#diff-3e17bfdb3630c1142bcaea49949d11342ed9f7edbf48121ff4ac7f1ebcd356a6R497), and because `OSD` loads into a element with an `id`, it was loading into this one (which we don't want)